test/controllers/hosts_controller_test.rb

```diff
@@ -500,6 +500,28 @@ def setup_multiple_compute_resource
       set_session_user.merge(:user => users(:admin).id)
   end
 
+  test "find multiple hosts by filter query" do
+    setup_user_and_host "edit"
+    post :update_multiple_owner, { :search => "",
+      :owner => { :id => users(:one).id_and_type}},
+      set_session_user.merge(:user => users(:admin).id)
+    as_admin do
+      assert_equal users(:one).id_and_type, @host1.reload.is_owned_by
+      assert_equal users(:one).id_and_type, @host2.reload.is_owned_by
+    end
+  end
```

Comment:
Please add a test that has an actual search query and make sure only the matching hosts are selected.